python/mlc_chat/compiler/model/extern_op.py

```diff
@@ -116,3 +117,39 @@ def _prefill():
     output = output.permute_dims([0, 2, 1, 3])  #  [b, s, h, d]
     output = output.reshape([b, s, h_q * d])  # [b, s, h * d]
     return output
+
+
+def decode_matmul(
```

Comment:
nit: more specifically, should its name be `faster_transformer_dequantize_gemm`?